cmd/clusterctl/pkg/client/config_test.go

```diff
@@ -359,7 +357,6 @@ func Test_clusterctlClient_GetClusterTemplate(t *testing.T) {
 				providerType:    clusterctlv1.InfrastructureProviderType,
 				version:         "v3.0.0",
 				flavor:          "",
-				bootstrap:       "kubeadm",
 				path:            "config-kubeadm.yaml",
```

Comment:
Why is this still config-kubeadm?

Reply:
the path variable is a leftover from some previous refactors. Sorry
Now I cleaned up everything